<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stuSqlDialog</class>
 <widget class="QDialog" name="stuSqlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#stuSqlDialog{
background-image: url(:/image/2.jpg);}

QDialog#stuSqlDialog QPushButton{
  border:none;
  border-radius:10px;
  background-color: rgb(255, 255, 255);
  color: rgb(0, 0, 0);
  font: 8pt &quot;Microsoft YaHei UI&quot;;   

}
 QDialog#stuSqlDialog  QPushButton:hover{
  
	background-color: rgb(177, 255, 255);
}
 QDialog#stuSqlDialog QLabel{
	color: rgb(0, 0, 0);
	font: 8pt &quot;Microsoft YaHei UI&quot;;
	background-color: rgb(255, 255, 255);
}
QDialog#stuSqlDialog QLineEdit{
	color: rgb(0, 0, 0);
	font: 10pt &quot;Microsoft YaHei UI&quot;;
	background-color: rgb(255, 255, 255);
}
</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>681</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>181</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>算法设计与分析</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>操作系统</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>大学英语</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>高等数学</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>名次</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QPushButton" name="BackBtn">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="numberLE"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>450</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="nameLE"/>
    </item>
    <item>
     <widget class="QLineEdit" name="suanfaLE"/>
    </item>
    <item>
     <widget class="QLineEdit" name="czxtLE"/>
    </item>
    <item>
     <widget class="QLineEdit" name="englishLE"/>
    </item>
    <item>
     <widget class="QLineEdit" name="mathLE"/>
    </item>
    <item>
     <widget class="QLineEdit" name="mingcile"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>651</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="UpSortBtn">
      <property name="text">
       <string>升序</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DownSortBtn">
      <property name="text">
       <string>降序</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AddBtn">
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SearchBtn">
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DelBtn">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>91</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
